docs/turn-howto.md

```diff
@@ -9,199 +9,19 @@ allows the homeserver to generate credentials that are valid for use on the
 TURN server through the use of a secret shared between the homeserver and the
 TURN server.
 
-The following sections describe how to install [coturn](<https://github.com/coturn/coturn>) (which implements the TURN REST API) and integrate it with synapse.
+This documentation provides two TURN server configuration examples:
+
+* [coturn](setup/turn/coturn.md)
+* [eturnal](setup/turn/eturnal.md)
 
 ## Requirements
 
-For TURN relaying with `coturn` to work, it must be hosted on a server/endpoint with a public IP.
+For TURN relaying with `eturnal` to work, it must be hosted on a server/endpoint with a public IP.
```

Comment:
Is a public IP address no longer required for `coturn`? Is there a differenz between `coturn` and `eturnal`?

Reply:
No, difference. Relay IP is needed for both. That was a copy paste error last night. I fix it in the next commit. thanks.